2.4/README.md

```diff
@@ -25,6 +25,9 @@ The following environment variables influence the MongoDB configuration file. Th
 |  `MONGODB_NOPREALLOC` | Disable data file preallocation.                                          |  true
 |  `MONGODB_SMALLFILES` | Set MongoDB to use a smaller default data file size.                      |  true
 |  `MONGODB_QUIET`      | Runs MongoDB in a quiet mode that attempts to limit the amount of output. |  true
+|  `MONGODB_TEXT_SEARCH_ENABLED` | Enables the [text search](https://docs.mongodb.org/v2.4/core/index-text/#text-search-text-command) feature.  |  false
+
+**Notice: The `MONGODB_TEXT_SEARCH_ENABLED` option is deprecated in MongoDB 2.6.**
```

Comment:
@omron93 my point was that our 2.6 image doesn't respect the MONGODB_TEXT_SEARCH_ENABLED env var, so the readme should not imply that it does  (deprecated implies supported/functional)

so what it seems like we should/will have:
2.4 - default(from mongo itself) is off, env variable to turn it on
2.6 - default(from mongo itself) is on, no option to turn it off currently.